packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -265,7 +270,9 @@ export function captureObserverEntries(list, { isHardNavigation, trStart }) {
    * To avoid capturing the unload event handler effect
    * as part of the page-load transaction duration
    */
-  const unloadDiff = timing.fetchStart - timing.navigationStart
+  const unloadDiff = isRedirectInfoAvailable(timing)
```

Comment:
this makes sure the FCP startTime includes the `redirectTime`, otherwise the mark would show up in the bad place in the UI